e2e/pages/match/searchPage.ts

```diff
@@ -33,7 +32,7 @@ export class SearchScreen extends BasePage {
     apType: ApTypeLabel
     preferredPostcode: string
   }): void {
-    this.shouldShowDatesOfPlacement(datesOfPlacement, duration)
+    super.shouldShowDatesOfPlacement(datesOfPlacement, duration)
```

Comment:
Another use of `super` instead of `this` I would get rid of!

Also, all the methods in `shouldShowApplicationDetails` are Promises, so they should all be `await`-ed and shouldShowApplicationDetails` should be async (aware this wasn't your code but let's fix if we can!)

Reply:
sure, no worries